adapters/remote-agent/src/nemo_fabric_adapters/remote_agent/adapter.py

```
@@ -0,0 +1,405 @@
+            headers: dict[str, str] = {}
+            if model.api_key_env is not None:
+                credential = os.environ[model.api_key_env]
+                if self._api_type == "anthropic-messages":
+                    headers["x-api-key"] = credential
+                else:
+                    headers["authorization"] = f"Bearer {credential}"
+            if self._api_type == "anthropic-messages":
+                headers["anthropic-version"] = "2023-06-01"
+
+            self._endpoint = _api_url(base_url, self._api_type)
+            self._client = httpx.AsyncClient(headers=headers, timeout=None)
```

Ruff (0.16.2)

[error] Probable use of `httpx` call with timeout set to `None`

(S113)